private/buf/cmd/buf/buf.go

```diff
@@ -180,14 +182,14 @@ func NewRootCommand(name string) *appcmd.Command {
 									repositoryundeprecate.NewCommand("undeprecate", builder),
 								},
 							},
-							//{
-							//	Use:   "branch",
-							//	Short: "Repository branch commands.",
-							//	SubCommands: []*appcmd.Command{
-							//		branchcreate.NewCommand("create", builder),
-							//		branchlist.NewCommand("list", builder),
-							//	},
-							//},
+							{
+								Use:   "track",
+								Short: "Repository track commands.",
+								SubCommands: []*appcmd.Command{
+									tracklist.NewCommand("list", builder),
+									trackdelete.NewCommand("delete", builder),
```

Comment:
Will we ever need a `create` command (would be in a follow-up, if ever)? Is it possible to create a track without a commit (i.e. instead of using the `buf push --track` command)? Or is that not something we plan to support?

Reply:
When we met about this @bufdev said there should be just one way to create a track, and I agree. It shouldn't be hard to add later if we find a compelling use case for it. As you pointed out, we will need to think through how to handle tracks with zero commits.